myshop/tests/cart/test_views.py

```diff
@@ -18,21 +18,37 @@ def setUp(self):
         )
 
     def test_cart_add_redirects_to_cart_detail(self):
-        self.cart_add_url = reverse(
-            "cart:cart_add", kwargs={"product_id": "1"}
-        )
-        self.cart_detail_url = reverse("cart:cart_detail")
-        response = self.client.post(self.cart_add_url)
+        cart_add_url = reverse("cart:cart_add", kwargs={"product_id": "1"})
+        cart_detail_url = reverse("cart:cart_detail")
+        response = self.client.post(cart_add_url)
+
+        self.assertEqual(cart_detail_url, response.url)
 
-        self.assertEqual(self.cart_detail_url, response.url)
+    def test_cart_remove_redirects_to_cart_detail(self):
+        cart_remove_url = reverse("cart:cart_add", kwargs={"product_id": "1"})
+        cart_detail_url = reverse("cart:cart_detail")
+        response = self.client.post(cart_remove_url)
+
+        self.assertEqual(cart_detail_url, response.url)
 
     def test_templates_used_for_cart_detail_view(self):
-        self.cart_detail_url = reverse("cart:cart_detail")
-        response = self.client.get(self.cart_detail_url)
+        cart_detail_url = reverse("cart:cart_detail")
+        response = self.client.get(cart_detail_url)
```

Comment:
Shouldn't be part of this PR. You can change in the PR when this code is added if the PR is still not merged or you can directly push this change to master and rebase this branch

Reply:
Resolved by refactoring the code in PR where the code was added